capactio/capact
# Fix storage backend ID for TypeInstance from nested workflow

## Description

Changes proposed in this pull request:
- fix storage backend ID for TypeInstance from nested workflow

There was incorrect backend mapping - we find it by the name of output Type Instance relation which is ok where there is no nested workflow. In the nested workflow, we need to find it by the name of the artifact.

## Testing

1. Checkout this PR
2. Create cluster using `DISABLE_MONITORING_INSTALLATION=true USE_TEST_SETUP=true make dev-cluster`
3. Change manifests source to my branch with no backend for psql:
```
kubectl set env deploy/capact-hub-public -n capact-system -c hub-public-populator MANIFESTS_SOURCES="github.com/mkuziemko/hub-manifests?ref=lost_bid"
```
4. Install Helm Storage and [Mattermost](https://capact.io/docs/example/mattermost-installation/)
5. Verify uploaded TypeInstaces. 

## Related issue(s)
- https://github.com/capactio/capact/issues/705
